app/views/admin/tracks/index.html.haml

```diff
@@ -11,29 +11,55 @@
         %th Name
         %th Short name
         %th Description
+        %th Submitter
         %th Color
+        %th State
+        %th Included in the Cfp
         %th Actions
       %tbody
         - @tracks.each do |track|
           %tr
             %td
-              = link_to(admin_conference_program_track_path(@conference.short_title, track.short_name)) do
+              = link_to(admin_conference_program_track_path(@conference.short_title, track)) do
                 = track.name
             %td
               = track.short_name
             %td
               %p
                 = truncate(track.description)
             %td
+              - if track.self_organized?
+                = link_to track.submitter.name, admin_user_path(track.submitter)
+              - else
+                N/A
+            %td
               %span.label{style: "background-color: #{track.color}; color: #{ contrast_color(track.color) }"}
                 = track.color
             %td
+              - if track.self_organized?
```

Comment:
We could always be showing the state, and mark as accepted/confirmed the tracks that are created on admin side